src/test/java/seedu/duke/commands/BalanceCommandTest.java

```diff
@@ -89,37 +89,36 @@ public void getBalance_ifCurrencyIsNotSpecified_shouldReturnAllCurrencies() {
                     "ALL",
                     accounts
             );
-            Account CNYAccount = null;
-            Account EURAccount = null;
-            for(Account account : output){
-                if(account.getCurrencyType().equals(Currency.EUR)){
-                    EURAccount = account;
-                }
-                else if(account.getCurrencyType().equals(Currency.CNY)){
-                    CNYAccount = account;
+            Account cnyAccount = null;
+            Account eurAccount = null;
+            for (Account account : output) {
+                if (account.getCurrencyType().equals(Currency.EUR)) {
+                    eurAccount = account;
+                } else if (account.getCurrencyType().equals(Currency.CNY)) {
+                    cnyAccount = account;
                 }
             }
-            if(CNYAccount == null || EURAccount == null){
+            if (cnyAccount == null || eurAccount == null) {
                 fail();
             }
-            System.out.println(CNYAccount.getBalance());
-            System.out.println(EURAccount.getBalance());
+            System.out.println(cnyAccount.getBalance());
+            System.out.println(eurAccount.getBalance());
```

Comment:
Do we need the `System.out.println` in the test? If we are testing what is printed on the screen, we can follow what Arif has done on the other screen?

Reply:
I didn't want to change the logic, @Arif-Khalid can you comment on this

Reply:
Noted, closing this